lib/fast_gettext/translation_repository/yaml.rb

```diff
@@ -51,6 +52,7 @@ def current_translations
       # Given a yaml file return a hash of key -> translation
       def load_yaml(file, locale)
         yaml = YAML.load_file(file)
+        raise Exceptions::InvalidYamlFormat.new(file) unless yaml.key?(locale)
         yaml_hash_to_dot_notation(yaml[locale])
```

Comment:
simply do a `yaml.fetch(locale)` and be done with it ? :D

Reply:
before this would have been nil ... so it would have blow up ... so minor bump should be good enough ?